### ISSUE TITLE
Filenames with backward slashes are unable to be downloaded

### ISSUE THREAD

**Issue by CyberFoxar** [yusuf8ahmed/Ship#5]
Hello, love the app concept.

However, on windows there's a problem with the downloading of files, related to how to filepaths are handled both on the web browser and in internals. 

On windows, filepaths begins with a letter, and not a forward slash. This confuses your templates and links become invalid.

Example:
Say our file is at this path in windows: `E:/A-folder/picture.jpg`
This is a valid Powershell-style path, as well as a pythonic one (though it is not a valid CMD one, that would be `E:\A-folder\picture.jpg`, which also has problems not related to this issue)
I then invoke ship using: `ship -o 'E:/A-folder/picture.jpg'`. Ship launches, all good. However, in the resulting HTML page, the preview, as well as the link don't work, because they're not valid HTML root-relative links !
Here's what you see in the HTML: 
```HTML
<button id="shipbutton">
        <a href="E:/A-folder/picture.jpg" style="text-decoration: none; color: white;" download="">
            <p>Ship It</p>
        </a>
</button>
```
While the `href` should begin with a forward slash `/`, i.e. `/E:/A-folder/picture.jpg`, which is a valid link that downloads the picture as expected.

&#8593; comment on issue #3

Hello again,

Another issue I hinted at in #3 . 

In windows, it is common to have filepaths looking like this: `E:\A-folder\picture.jpg`. This is a valid path in both Powershell and CMD.
So, invoking ship like so `ship -o 'E:\A-folder\picture.jpg'` does not yield any errors, because `os.path` see it as a valid path. And the page shown by ship even has the right link (though there is still the issue discussed in #3 ) : 
```html
<a href="E:\A-folder\picture.jpg" style="text-decoration: none; color: white;" download="">
            <p>Ship It</p>
        </a>
```

However, when trying to follow `/E:\A-folder\picture.jpg` (a technically valid URL, even if it's weird), it seems all backslashes are converted into forward slashes and I'm redirected to `/E:/A-folder/picture.jpg`, which ship doesn't accept, as it's not equal to the filename.

My advice would be to forgo using `os.path` to handle windows paths altogether and use `pathlib` (a standard, recent library) to handle them.

**Comment by yusuf8ahmed** [comment]
Thanks for opening the issue, This bug was fixed and should be in release alpha 0.0.4.0, but a correction should be 
#3 #5

**Comment by yusuf8ahmed** [comment]
Thanks for opening the issue, This bug was fixed and should be in release alpha 0.0.4.0 #3 #5
